src/lib.rs

```diff
@@ -306,6 +325,21 @@ where
         let bufs = &mut *guard;
         let mut event_buf = &mut bufs.current_buf;
 
+        // Time.
+
+        {
+            let prev_buffer_len = event_buf.len();
+
+            self.timer
+                .format_time(&mut event_buf)
+                .expect("Unable to write time to buffer");
+
+            // Something was written to the buffer, pad it with a space.
+            if prev_buffer_len < event_buf.len() {
+                write!(event_buf, " ").expect("Unable to write to buffer");
+            }
+        }
```

Comment:
UI test was failing because of the extra padding when no timestamp was actually written, so I came up with this workaround. Is this too hacky?